layouts/partials/footer.html

```diff
@@ -1,12 +1,15 @@
+{{- $title          := .Site.Params.title }}
 {{- $footer      := .Site.Params.footer }}
 {{- $logo        := index $footer "logo" }}
 {{- $quickLinks  := index $footer "quicklinks" }}
 {{- $socialMedia := index $footer "socialmedia" }}
+{{- $navbarLogo     := .Site.Params.navbarlogo }}
```

Comment:
Shall we keep the capitalization constant throughout?

Reply:
I will do that in a followup PR, but I would first like to test this with numpy.org.